stix2elevator/options.py

```diff
@@ -142,6 +142,9 @@ def __init__(self, cmd_args=None, file_=None, incidents=False,
             self.markings_allowed = cmd_args.markings_allowed
             if hasattr(cmd_args, "output_directory"):
                 self.output_directory = cmd_args.output_directory
+            # validator arg --silent is currently broken
+            # if self.silent:
+            #    self.validator_args += " --silent"
```

Comment:
I think we can remove these comments. (Line 145-147)

Reply:
I put this in to remind me - but perhaps it would be confusing to others?